sentry/sentry.conf.example.py

```diff
@@ -317,8 +317,7 @@ def get_internal_network():
 # OpenAI API key to turn on the feature.
 OPENAI_API_KEY = env("OPENAI_API_KEY", "")
 
-if OPENAI_API_KEY:
-  SENTRY_FEATURES["organizations:open-ai-suggestion"] = True
+SENTRY_FEATURES["organizations:open-ai-suggestion"] = bool(OPENAI_API_KEY)
```

Comment:
I'd prefer this to be just:
```python
SENTRY_FEATURES["organizations:open-ai-suggestion"] = False
```
So users will have to be aware that they're toggling the OpenAI feature. No billing surprise because of their Sentry deployments.

Reply:
But would you argue that only users who would want the OpenAI feature would be setting the `OPENAI_API_KEY`?

Reply:
@hubertdeng123 Pretty much yeah, OpenAI billing is so scary when you're forget that you have to pay for them